openlr/candidate_points_getter.cpp

```diff
@@ -0,0 +1,100 @@
+#include "openlr/candidate_points_getter.hpp"
+
+#include "openlr/helpers.hpp"
+
+#include "routing/road_graph.hpp"
+
+#include "storage/country_info_getter.hpp"
+
+using namespace routing;
+
+namespace
+{
+MwmSet::MwmId GetMwmIdByPoint(Index const & index, m2::PointD const & p)
+{
+  // TODO(mgsergio): Cache it!
+  auto const infoGetter =
```

Comment:
s/instad/instead/
Please, use it right now. And cache it.

Reply:
Done.